Networking/Repository/AbstractRepository.swift

```diff
@@ -133,18 +133,25 @@ open class AbstractRepository {
     fileprivate let _sessionManager: SessionManagerType
     fileprivate let _requestExecutor: RequestExecutorType
     
+    fileprivate let _defaultHeaders: [String: String]?
```

Comment:
It expects `[String: String]`: 

`Alamofire` defines: `public typealias HTTPHeaders = [String : String]`